src/components/common/CircularProgressWithPath.tsx

```diff
@@ -48,34 +57,31 @@ export const CircularProgressWithPath = ({
     <Box sx={{ position: 'relative', display: 'inline-flex' }}>
       <CircularProgress
         variant="determinate"
-        value={circularValue}
+        value={circularValue >= minPercent ? circularValue : minPercent}
         size={size}
         thickness={thickness}
         sx={progressSx}
         color={color}
-        data-cy={NUMBER_OF_ATTEMPTS_CIRCULAR_PROGRESSION_CY}
+        className={className}
       />
-      <Box
-        sx={{
-          top: 0,
-          left: 0,
-          bottom: 0,
-          right: 0,
-          position: 'absolute',
-          display: 'flex',
-          alignItems: 'center',
-          justifyContent: 'center',
-        }}
-      >
-        <Typography
-          variant="caption"
-          component="div"
-          color="text.secondary"
-          data-cy={NUMBER_OF_ATTEMPTS_CIRCULAR_PROGRESSION_TEXT_CY}
+      {showText && (
+        <Box
+          sx={{
+            top: 0,
+            left: 0,
+            bottom: 0,
+            right: 0,
```

Comment:
It is mandatory to have the 4 edges to center the text in the middle of the circular progress. But because for now the text is never used, I will just remove it to simplify the code.